.evergreen/evergreen.yml

```diff
@@ -134,12 +134,13 @@ buildvariants:
 
   - name: generator
     display_name: Generator
-    allowed_requesters: ["github_pr"]
+    patch_only: true
     # Omit node_version expansion so that yarn-install fails to save time.
     run_on:
       - ubuntu2204-small
     tasks:
       - generate-tasks
+      - generate-spruce-e2e
```

Comment:
I think at some point during this PR review the generator task may have broken. I don't see an e2e task being generated for spruce on the [latest run as of this comment](https://spruce.mongodb.com/version/66f306cb20dd7f00073dceb2/tasks?page=0&sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC&statuses=success&variant=%5Espruce%24) but it looks like it was working [before](https://evergreen.mongodb.com/build/evergreen_ui_spruce_patch_81659d0bf49cbb476b673a093faa803ccdfef537_66f16e4848333100071123ab_24_09_23_13_34_01?redirect_spruce_users=true). (Just wanted to flag in case it wasn't intentional)

Reply:
it's because it needs to be added to the project settings, since it hasn't been added yet no e2e_parallel task will be generated